linkup-cli/install.py

```diff
@@ -203,11 +204,15 @@ def download_and_extract(user_os: OS, user_arch: Arch, channel: Channel, release
 
     print(f"Decompressing {local_tar_path}")
     with tarfile.open(local_tar_path, "r:gz") as tar:
-        tar.extractall(path="/tmp")
+        tar.extractall(path="/tmp", filter="data")
 
     LINKUP_BIN_PATH.mkdir(parents=True, exist_ok=True)
-    shutil.move("/tmp/linkup", LINKUP_BIN_PATH / "linkup")
-    os.chmod(LINKUP_BIN_PATH / "linkup", 0o755)
+    linkup_bin_path = LINKUP_BIN_PATH / "linkup"
+    shutil.move("/tmp/linkup", linkup_bin_path)
+    os.chmod(linkup_bin_path, 0o755)
+
+    if user_os.Linux:
+        subprocess.run(["sudo", "setcap", "cap_net_bind_service=+ep", f"{linkup_bin_path}"])
```

Comment:
@RayProud this should be:
```suggestion
    if user_os == OS.Linux:
        subprocess.run(["sudo", "setcap", "cap_net_bind_service=+ep", f"{linkup_bin_path}"])
```